defaults/main.yml

```diff
@@ -0,0 +1,11 @@
+---
+urllib3_version: "1.21.*"
+pyOpenSSL_version: "17.0.*"
+ndghttpsclient_version: "0.4.*"
+pyasn1_version: "0.2.*"
+
+python_security_packages:
```

Comment:
I worry a bit about this as a top level role variable. Having it doesn't seem to buy us much benefit, and keeping it around feels like we're opening up access to a private API.